seatunnel-formats/seatunnel-format-avro/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.seatunnel</groupId>
+        <artifactId>seatunnel-formats</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>seatunnel-format-avro</artifactId>
+    <name>SeaTunnel : Formats : Avro</name>
+
+    <properties>
+        <avro.version>1.10.2</avro.version>
```

Comment:
How is the dependent version defined? Why use this version? If I use hudi sink, can this format be used?

Reply:
change to 1.9.0. 
since 1.9.0 use java LocalDate and Instant as date, timestamp type. before this version avro used joda time related api.

this format complete the convert between SeaTunnelRow and Avro GenericRecord (see RowToAvroConverter and AvroToRowConverter). so in general it can be used of any source and sink.